test/contour_tracing.jl

```diff
@@ -0,0 +1,85 @@
+
+@testset "contour_tracing" begin
+
+    # Call contour tracing algorithm and compare the number of components

```

Comment:
We could wrap all of these tests with a for loop such as:
```julia
        for T in (Int, Bool, Gray{Bool}, Gray{N0f8}, Gray{N0f16}, Gray{Float32}, Gray{Float64})
              labels = label_components(ContourTracing(), T.(test_image_1))
              num_components = maximum(labels)
              @test num_components == 0
              #  etc.
        end
```

This will verify that all the standard types that we can expect are properly handled.